geometric_kernels/spaces/circle.py

```diff
@@ -23,14 +24,14 @@ class SinCosEigenfunctions(EigenfunctionWithAdditionTheorem):
     """
 
     def __init__(self, num_eigenfunctions: int) -> None:
-        assert (
-            num_eigenfunctions % 2 == 1
-        ), "num_eigenfunctions needs to be odd to include all eigenfunctions within a level."
-        assert num_eigenfunctions >= 1
+        # assert (
```

Comment:
I did not get why we do not need this assertion anymore, is it because we now have it in `_addition_theorem`